src/ekore/anomalous_dimensions/unpolarized/time_like/as2.py

```diff
@@ -0,0 +1,741 @@
+                    + 2564 * npp(N, 4)
+                    + 379 * npp(N, 5)
+                    - 1177 * npp(N, 6)
+                    - 1037 * npp(N, 7)
+                    - 423 * npp(N, 8)
+                    - 76 * npp(N, 9)
+                    - 288 * zeta2 * npp(N, 2)
+                    - 720 * zeta2 * npp(N, 3)
+                    - 504 * zeta2 * npp(N, 4)
+                    + 180 * zeta2 * npp(N, 5)
+                    + 576 * zeta2 * npp(N, 6)
+                    + 504 * zeta2 * npp(N, 7)
+                    + 216 * zeta2 * npp(N, 8)
+                    + 36 * zeta2 * npp(N, 9)
+                    - 180 * m1tpN * npp(N, 3)
+                    - 72 * m1tpN * npp(N, 4)
+                    + 108 * m1tpN * npp(N, 5)
+                    + 108 * m1tpN * npp(N, 6)
+                    + 36 * m1tpN * npp(N, 7)
+                )
+            )
+            / (9 * (-1 + N) * npp(N, 3) * npp(1 + N, 3) * npp(2 + N, 3))
+            + (8 * sm2 * (2 + N + npp(N, 2))) / (N * (1 + N) * (2 + N))
+            + (
+                4
+                * s1
+                * (
+                    -48
+                    - 100 * N
+                    + 40 * npp(N, 2)
+                    + 77 * npp(N, 3)
+                    + 32 * npp(N, 4)
+                    + 11 * npp(N, 5)
+                )
+            )
+            / (3 * npp(N, 2) * npp(1 + N, 2) * npp(2 + N, 2))
+            - (4 * npp(s1, 2) * (2 + N + npp(N, 2))) / (N * (1 + N) * (2 + N))
+            + (12 * s2 * (2 + N + npp(N, 2))) / (N * (1 + N) * (2 + N))
+        )
+    )
+    qg3 = (
+        nf
+        * nf
+        * (
+            (
+                8
+                * (
+                    -12
+                    - 16 * N
+                    + 37 * npp(N, 2)
+                    + 41 * npp(N, 3)
+                    + 17 * npp(N, 4)
+                    + 5 * npp(N, 5)
+                )
+            )
+            / (9 * npp(N, 2) * npp(1 + N, 2) * npp(2 + N, 2))
+            - (8 * s1 * (2 + N + npp(N, 2))) / (3 * N * (1 + N) * (2 + N))
+        )
+    )
+    result = (1 / (2 * nf)) * (qg1 + qg2 + qg3)
+    return result
+
+
+@nb.njit(cache=True)
+def gamma_gq(N, nf):
+    r"""Compute the NLO gluon-quark anomalous dimension.
+
+    Implements Eqn. (B.11) from :cite:`Mitov:2006wy`
+    and Eqn. (A1) from :cite:`Gluck:1992zx`.
+
+    Parameters
+    ----------
+    N : complex
+        Mellin moment
+    nf : int
+        No. of active flavors
+    cache : numpy.ndarray
+        Harmonic sum cache
+    is_singlet : boolean
+        True for singlet, False for non-singlet, None otherwise
+
+    Returns
+    -------
+    gamma_gq : complex
+        NLO gluon-quark anomalous dimension
+        :math:`\gamma_{gq}^{(1)}(N)`
+
+    """
+    s1 = w1.S1(N)  # c.get(c.S1, cache, N, is_singlet)
+    s2 = w2.S2(N)  # c.get(c.S2, cache, N, is_singlet)
+    sm2 = w2.Sm2(N, s2, True)  # c.get(c.Sm2, cache, N, is_singlet)
+
+    m1tpN = 1
```

Comment:
Since it's 1, you can also drop it. I hope it will be optimized away, but at least it would uselessly increase compile-time.